source/install/upgrading-rdo-1.html.adoc

```diff
@@ -112,17 +112,66 @@ Upgrade the database schema for each service that uses the database. Run the fol
 |Image Service |glance |`# su -s /bin/sh -c "glance-manage db_sync" glance`
 |Block Storage |cinder |`# su -s /bin/sh -c "cinder-manage db sync" cinder`
 |Orchestration |heat |`# su -s /bin/sh -c "heat-manage db_sync" heat`
-|Compute |nova |
-`# su -s /bin/sh -c "nova-manage api_db sync" nova`
-
-`# su -s /bin/sh -c "nova-manage db sync" nova`
+|Compute |nova | See xref:upgrading-compute-all-at-once[Upgrading Compute (nova)].
 |Telemetry |ceilometer |`# ceilometer-dbsync`
 |Telemetry Alarming |aodh |`# aodh-dbsync`
 |Telemetry Metrics |gnocchi |`# gnocchi-upgrade`
 |Clustering |sahara | `# su -s /bin/sh -c "sahara-db-manage upgrade heads" sahara`
 |Networking |neutron |`# su -s /bin/sh -c "neutron-db-manage upgrade heads" neutron`
 |===
 
+[[upgrading-compute-all-at-once]]
+### Upgrading Compute (nova)
+
+In Ocata, the Compute service requires at least two Cells V2 cell databases, the fist database is always called `cell0` and the second database can be named arbitrarily, for example, `cell1`.
```

Comment:
Cell0 isn't really a cell, it's a placeholder for failed instances. Might be better to say that Ocata needs at least one Cell v2 cell database and the cell0 database.